### PR TITLE
Tailwind CSS #465

### PR DESCRIPTION
**Pull request by Amit366**
Updated the changes suggested.

### CODE REVIEW

**Comment by smaranjitghose** [comment]
The deployment is failing. Could you please attach a clip of the changes

File: src/components/Header/Header.module.css [hunk 1]
```diff
@@ -92,28 +92,13 @@
...
 
+@media only screen and (max-width: 460px) {
     .right {
-        height: 35px;
-        padding-right: 20px;
-        overflow: hidden;
-        transform: translateY(0);
-        transition: all 0.1s cubic-bezier(0.9, 0, 0.33, 1);
-    }
-    .rightDrop{
```
Comment (smaranjitghose) [inline src/components/Header/Header.module.css:113]: How are you going to define the waves?

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import styles from './Header.module.css';
 import logo from './logo.png';
 import {ImInfo} from "react-icons/im";
 import {RiTeamLine, RiContactsBook2Line} from "react-icons/ri";
-import {BiDonateHeart, BiMenu} from "react-icons/bi";
```
Comment (smaranjitghose) [inline src/components/Header/Header.js:8]: Can you please explain the reason behind this change?

Comment (Amit366) [reply src/components/Header/Header.js:8]: Sir in tailwind there is no need for hamburger as it itself gets responsive so I made this changes.

**Comment by Amit366** [comment]
@smaranjitghose I don't know why is it failing as I am able to deploy it.

File: src/components/Header/Header.js (hunk 2 above)
Comment (smaranjitghose) [reply src/components/Header/Header.js:8]: What about the animation?😅

File: public/index.html [hunk 3]
```diff
@@ -17,6 +17,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/tailwindcss/1.5.1/tailwind.min.css" rel="stylesheet">
```
Comment (smaranjitghose) [inline public/index.html:20]: This directory should not touched

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
Do one change at a time. Your PR replaces a lot of things which are not necessary

**Pull request closed without merging**